examples/07-collaboration/04-comments/README.md

```diff
@@ -1,13 +1,18 @@
-# Comments & Threads
+# Threads Sidebar View
 
-In this example, you can add comments to the document while collaborating with others. You can also pick user accounts with different permissions, as well as react to, reply to, and resolve existing comments.
+This is another example showcasing threads and comments. However, instead of
```

Comment:
this text ("This is another example") doesn't work because you can't assume the user has seen the other example